### PR TITLE
test(devtools-view): Remove dead screenshot tests

### PR DESCRIPTION
**Pull request by Josmithr**
These tests have been dead for quite a while, and their existence creates more problems than potential value. We will come back and reconsider options for visual regression testing in the near future.

[AB#8120](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/8120)

### CODE REVIEW

**Comment by seanimam** [comment]
I would love to have a small post-mortem somewhere on what went wrong and how we could potentially solve the issues in the future

**Review by alexvy86 — APPROVED** [review]
I think we can also remove the `.gitattributes` file in the package? Otherwise lgtm.

**Review by seanimam — APPROVED** [review]

**Review by WayneFerrao — APPROVED** [review]
Agreed on the post-mortem.

**Review by jason-ha — APPROVED** [review]

File: packages/tools/devtools/devtools-view/src/test/tsconfig.cjs.json [hunk 1]
```diff
@@ -2,14 +2,11 @@
 	// This config must be used in a "type": "commonjs" environment. (Use fluid-tsc commonjs.)
```
Comment (jason-ha) [inline packages/tools/devtools/devtools-view/src/test/tsconfig.cjs.json:2]: Looks like comment should be kept. If it is used without `fluid-tsc commonjs`, then it generates ESM.

File: packages/tools/devtools/devtools-view/package.json [hunk 2]
```diff
@@ -152,31 +145,16 @@
...
+				"build:esnext"
 			],
 			"eslint": [
 				"tsc",
 				"build:esnext",
-				"build:test:jest:esm",
-				"build:test:mocha",
-				"build:test:utils:esm"
+				"build:test:esm"
 			]
```
[the remark this answers is not in the document]
Comment (Josmithr) [reply packages/tools/devtools/devtools-view/package.json:158]: Awesome. I'll remove the whole block then.

File: packages/tools/devtools/devtools-view/src/test/tsconfig.cjs.json (hunk 1 above)
Comment (Josmithr) [reply packages/tools/devtools/devtools-view/src/test/tsconfig.cjs.json:2]: I didn't remove it. It's still there, though github is highlighting the line in red for some reason.

**Comment by Josmithr** [comment]
> I think we can also remove the `.gitattributes` file in the package? Otherwise lgtm.

Good catch! Removed.

**Pull request merged**